components/omega/doc/userGuide/OmegaBuild.md

```diff
@@ -43,31 +68,32 @@ src
 test
 ```
 
-To build the Omega, execute the `make` command in the directory.
+To build the Omega, execute the `make` command in the build directory.
 
 ```sh
 >> make
 Scanning dependencies of target yakl
-[ 11%] Building Fortran object external/YAKL/CMakeFiles/yakl.dir/src/YAKL_gator_mod.F90.o
-[ 22%] Building CXX object external/YAKL/CMakeFiles/yakl.dir/src/YAKL.cpp.o
-[ 33%] Linking CXX static library libyakl.a
-[ 33%] Built target yakl
-[ 44%] Building CXX object src/CMakeFiles/OmegaLib.dir/ocn/ocndummy.cpp.o
-[ 55%] Linking CXX static library libOmegaLib.a
-[ 55%] Built target OmegaLib
-[ 66%] Building CXX object src/CMakeFiles/omega.exe.dir/drivers/drvdummy.cpp.o
-[ 77%] Linking CXX executable omega.exe
-[ 77%] Built target omega.exe
-[ 88%] Building CXX object test/CMakeFiles/OMEGA_TEST.dir/testdummy.cpp.o
-[100%] Linking CXX executable OMEGA_TEST
-[100%] Built target OMEGA_TEST
+[ 10%] Building Fortran object external/YAKL/CMakeFiles/yakl.dir/src/YAKL_gator_mod.F90.o
+[ 20%] Building CXX object external/YAKL/CMakeFiles/yakl.dir/src/YAKL.cpp.o
+[ 30%] Linking CXX static library libyakl.a
+[ 30%] Built target yakl
+[ 40%] Building CXX object src/CMakeFiles/OmegaLib.dir/base/MachEnv.cpp.o
+[ 50%] Building CXX object src/CMakeFiles/OmegaLib.dir/ocn/OcnDummy.cpp.o
+[ 60%] Linking CXX static library libOmegaLib.a
+[ 60%] Built target OmegaLib
+[ 70%] Building CXX object src/CMakeFiles/omega.exe.dir/drivers/DrvDummy.cpp.o
+[ 80%] Linking CXX executable omega.exe
+[ 80%] Built target omega.exe
+[ 90%] Building CXX object test/CMakeFiles/testDataTypes.exe.dir/base/DataTypesTest.cpp.o
+[100%] Linking CXX executable testDataTypes.exe
+[100%] Built target testDataTypes.exe
```

Comment:
I wouldn't bother to update this every time Omega changes.  I would suggest editing the docs to indicate that this is just an example of the output to expect.